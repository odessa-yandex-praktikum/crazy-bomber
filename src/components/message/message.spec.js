import React from 'react';
import renderer from 'react-test-renderer';
import {MessageItem} from './index';

it('renders correctly', () => {
    const message = {
        message: 'Hello, World',
        author: 'Admin',
        created: Date.parse('Sun, 16 May 2021 15:43:00'),
    };
    const tree = renderer.create(<MessageItem message={message} key={message.created} />).toJSON();
    expect(tree).toMatchSnapshot();
});
