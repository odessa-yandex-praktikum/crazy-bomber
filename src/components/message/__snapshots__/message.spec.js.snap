// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  className="forum-page__item"
>
  <div
    className="forum-page__topic-message"
  >
    <img
      alt="author_image"
      className="forum-page__author-image"
    />
    <p
      className="forum-page__discussion-message"
    >
      Hello, World
    </p>
  </div>
  <div
    className="forum-page__discussion-content_created"
  >
    <p
      className="forum-page__created-date"
    >
      Left 
      1 month ago
    </p>
  </div>
</li>
`;
